module/product/src/Application/Controller/Api/Bindings/ProductRemoveBindingAction.php

```diff
@@ -72,8 +80,16 @@ public function __construct(CommandBusInterface $commandBus)
      */
     public function __invoke(Language $language, AbstractProduct $product, AbstractAttribute $binding): Response
     {
-        $this->commandBus->dispatch(new RemoveProductBindingCommand($product, $binding));
+        $data = new ProductBindFormModel($product);
+        $data->bindId = $binding->getId()->getValue();
+        $constraint = new ProductHasChildren();
+        $violations = $this->validator->validate($data, $constraint);
```

Comment:
```suggestion
        $violations = $this->validator->validate($data);
```

The model is configured with the constraint, there's no need to pass it manually I think